### ISSUE TITLE
Accessibility: unlabeled buttons with talkback screen reader

### ISSUE THREAD

**Issue by shortwavesurfer2009** [ImranR98/Obtainium#2151]
**Prerequisites**

**Describe the bug**

When using the Talkback screen reader, the buttons shown in the screenshot below read as unlabeled.

**To Reproduce**

![Image](https://github.com/user-attachments/assets/929923e1-241c-49c5-8f10-3f91fdd03178)

**Please complete the following information:**
 - Device:  OnePlus Nord N200
 - OS:  LineageOS 21
 - Obtainium Version:  1.1.45

**Additional context**

**Comment by ImranR98** [comment]
Hmm, these buttons all seem to work fine on my phone (granted I don't use a screen reader but I was able to get it to navigate to those buttons via swiping and the TTS did say the tooltip descriptions).

**Comment by shortwavesurfer2009** [comment]
That's weird. They definitely did not work for me.

**Comment by aryanchoudharypro** [comment]
> Hmm, these buttons all seem to work fine on my phone (granted I don't use a screen reader but I was able to get it to navigate to those buttons via swiping and the TTS did say the tooltip descriptions).

I can confirm that these buttons don't get announced by TalkBack. Are the semantics labels added properly?

**Comment by Jonathans859** [comment]
I have the same issue. The buttons are not labeled e.g. spoken out by Talkback correctly. You can interact with them but you basically don't know what you're interacting with.

**Comment by aryanchoudharypro** [comment]
@ImranR98  is there any update regarding this issue

**Comment by AbdelazizHatem** [comment]
- [ ] Hello, we are facing issues with this application using TalkBack screen reader
. It's true that it is compatible with accessibility, but there are some problems when we click on the names of the applications. I will send you some examples with the names of the contracts so you can fix them in the next update. When I click on the name of the application to go to its details, it takes me to the change log. Please divide the focus items so that next to each application, it shows the change log or makes it an expandable list when the user clicks on it. These are the contracts that I will give you now related to one of the buttons available in the desired application. 
android.view.accessibility.AccessibilityNodeInfo@aa9d; boundsInParent: Rect(0, 0 - 1080, 189); boundsInScreen: Rect(0, 370 - 1080, 559); packageName: dev.imranr.obtainium; className: android.widget.ImageView; text: null; error: null; maxTextLength: -1; stateDescription: null; contentDescription: aShell You
بواسطة DP-Hridayan
v6.1.0
٢٠٢٥-٠٤-٢٣; tooltipText: null; viewIdResName: ; checkable: false; checked: false; focusable: true; focused: false; selected: false; clickable: true; longClickable: false; contextClickable: false; enabled: true; password: false; scrollable: false; importantForAccessibility: true; visible: true; actions: [AccessibilityAction: ACTION_CLICK - null, AccessibilityAction: ACTION_CLEAR_ACCESSIBILITY_FOCUS - null]
